camayoc/utils.py

```diff
@@ -19,10 +19,6 @@
 client_cmd = settings.quipucords_cli.executable
 """Client command to use during tests. Defaults to `qpc`."""
 
-client_cmd_name = settings.quipucords_cli.display_name
-"""Client name displayed on help texts. Defaults to `qpc`."""
-# this is useful when client_cmd is set to an absolute path, has extra arguments like -v
```

Comment:
Suggestion: drop all the other changes (except the one I pointed in other comment) and keep refactor `client_cmd_name` here to infer CLI name somehow. Mirek gave an excellent suggestion using pathlib, but feel free to do whatever you see fit.